### PR TITLE
test: Updated index tests

### PR DESCRIPTION
**Pull request by richburdon**
WIP

### CODE REVIEW

**Comment by mykola-vrmchk** [comment]
* **#6757** <a href="https://app.graphite.dev/github/pr/dxos/dxos/6757?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @richburdon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by mykola-vrmchk — APPROVED** [review]

File: packages/sdk/client/src/tests/indexing.test.ts [hunk 1]
```diff
@@ -236,8 +260,9 @@ describe('Index queries', () => {
     const client = await initClient(builder.createLocalClientServices());
     await client.halo.createIdentity();
     const space = await client.spaces.create();
-    const contact = await addContact(space, john);
-    const document = await addDocument(space, 'important document');
+
+    const [contact] = await addObjects(space, getObjects().contacts);
```
Comment (richburdon) [inline packages/sdk/client/src/tests/indexing.test.ts:264]: this generates all of the data every time.
better to generate the data once -- and generate from static arrays of RAW object.
but at least
```
const { contacts, documents } = createObjects();
```

File: packages/sdk/client/src/tests/indexing.test.ts [hunk 2]
```diff
@@ -253,13 +278,25 @@ describe('Index queries', () => {
...
+    await addObjects(space, getObjects().expandos);
     const query = space.db.query(Filter.schema(Expando));
-    await query.run();
-
-    await checkIfQueryContainsObject(space.db.query(Filter.schema(Expando)), Expando, { data: { name: john } });
+    await matchObjects(query, getObjects().expandos);
   });
 });
+
+const objectContains = (container: Object, content: Object): Boolean => {
```
Comment (richburdon) [inline packages/sdk/client/src/tests/indexing.test.ts:287]: rename matchObjects?
and `any` not `Object`?

File: packages/sdk/client/src/tests/indexing.test.ts [hunk 3]
```diff
@@ -29,56 +66,44 @@ describe('Index queries', () => {
...
       { fire: true },
     );
-    const obj = await receivedIndexedObject.wait({ timeout: 5000 });
-    for (const key of Object.keys(content)) {
-      expect(obj[key]).to.deep.equal(content[key]);
-    }
-    return obj;
+
+    const queriedObjects = await receivedIndexedObject.wait({ timeout: TIMEOUT });
+    unsubscribe();
```
Comment (richburdon) [inline packages/sdk/client/src/tests/indexing.test.ts:105]: can you just `await query.run`?

File: packages/core/echo/echo-schema/src/expando.ts [hunk 4]
```diff
@@ -6,13 +6,13 @@ import * as S from '@effect/schema/Schema';
 
 import { echoObject } from './annotations';
 
-export const ExpandoTypename = 'Expando';
+export const EXPANDO_TYPENAME = 'Expando';
```
Comment (richburdon) [inline packages/core/echo/echo-schema/src/expando.ts:9]: TYPE_EXPANDO = `dxos.org/type/Expando`
but i think we need to discuss typenames w/Dima.

Comment (mykola-vrmchk) [reply packages/core/echo/echo-schema/src/expando.ts:9]: This is braking change

**Pull request merged**